packages/dapp/src/components/Widgets/Widgets.style.tsx

```diff
@@ -74,15 +74,18 @@ export const WidgetContainer = styled(Box, {
     transitionProperty: 'margin-top, padding-top, transform',
     transitionDuration: '.3s',
     transitionTimingFunction: 'ease-in-out',
-    marginTop: showWelcome ? 'calc( 10% - 24px )' : '0',
+    marginTop: showWelcome ? '24px' : '0',
 
     [`@media screen and (min-height: 700px)`]: {
-      marginTop: showWelcome ? 'calc( 50vh - 90%)' : '0',
+      marginTop: showWelcome ? 'calc( 50vh - 680px / 2.5 - 80px)' : '0',
```

Comment:
Can we avoid this magic calculation somehow? 😅

Reply:
@chybisov 
I wish I knew how :D Please, if you have any idea on how to fix this, share it with me 

![image](https://github.com/lifinance/jumper.exchange/assets/43956540/5ad8a275-5a2e-449c-a7d8-b9c2330f8135)

Reply:
This is the result on my mobile:
![image](https://github.com/lifinance/jumper.exchange/assets/43956540/d775c0b4-23d3-4f9a-9323-3ff5c648efe8)

Reply:
On a landscape mode on mobile, the widget is not shown:

![image](https://github.com/lifinance/jumper.exchange/assets/43956540/90c13cf2-529e-4464-928c-2cd8056a127a)

Only, by disabling the marginTop of the widget (which adds space between the navigation and itself) and also disabling the paddingTop (which allows the widget to react to the hover effect, moving up into the screen), the widget would slightly be visible.

In my opinion, it should be fine? There is little space anyways :D